sw/device/tests/BUILD

```diff
@@ -3459,3 +3459,44 @@ opentitan_binary(
         "//sw/device/silicon_creator/manuf/lib:sram_start",
     ],
 )
+
+[
+    opentitan_test(
+        name = "uart{}_tx_rx_test".format(uart_idx),
+        srcs = ["uart_tx_rx_test.c"],
+        copts = ["-DUART_IDX={}".format(uart_idx)],
+        cw310 = new_cw310_params(
+            bitstream = "//hw/bitstream/hyperdebug:test_rom",
```

Comment:
I think with the new rules, we are supposed to avoid using `bitstream` and use `otp`. See [the sival dev guide](https://github.com/lowRISC/opentitan/blob/master/sw/device/tests/doc/sival/devguide.md)

Reply:
For now I have changed the manually specified bitstream to `//hw/bitstream/hyperdebug:rom_with_fake_keys_otp_prod` which (I think) uses the bitstream built in CI. This is messing with the new execution environment / splicing rules, though, and should be updating ASAP.